homeassistant/components/simplisafe/lock.py

```diff
@@ -38,6 +38,11 @@ def __init__(self, simplisafe, system, lock):
         for event_type in (EVENT_LOCK_LOCKED, EVENT_LOCK_UNLOCKED):
             self.websocket_events_to_listen_for.append(event_type)
 
+    @property
```

Comment:
Why? The `SimpliSafeEntity` class from which the `SimpliSafeLock` class inherits already has a unique ID defined.

Reply:
Fair enough